{"error":{"name":"MongoError","message":"connection 1 to app.parentplanet.com:27017 timed out"},"level":"error","message":"Error generating response. { MongoError: connection 1 to app.parentplanet.com:27017 timed out\n    at Function.MongoError.create (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/error.js:29:11)\n    at Socket.<anonymous> (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/connection/connection.js:161:20)\n    at Socket.g (events.js:286:16)\n    at emitNone (events.js:86:13)\n    at Socket.emit (events.js:185:7)\n    at Socket._onTimeout (net.js:333:8)\n    at tryOnTimeout (timers.js:224:11)\n    at Timer.listOnTimeout (timers.js:198:5)\n  name: 'MongoError',\n  message: 'connection 1 to app.parentplanet.com:27017 timed out' }","timestamp":"2016-08-10T05:51:08.680Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: connection 1 to app.parentplanet.com:27017 timed out\n    at Function.MongoError.create (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/error.js:29:11)\n    at Socket.<anonymous> (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/connection/connection.js:161:20)\n    at Socket.g (events.js:286:16)\n    at emitNone (events.js:86:13)\n    at Socket.emit (events.js:185:7)\n    at Socket._onTimeout (net.js:333:8)\n    at tryOnTimeout (timers.js:224:11)\n    at Timer.listOnTimeout (timers.js:198:5)\n  name: 'MongoError',\n  message: 'connection 1 to app.parentplanet.com:27017 timed out' } MongoError: connection 1 to app.parentplanet.com:27017 timed out\n    at Function.MongoError.create (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/error.js:29:11)\n    at Socket.<anonymous> (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/connection/connection.js:161:20)\n    at Socket.g (events.js:286:16)\n    at emitNone (events.js:86:13)\n    at Socket.emit (events.js:185:7)\n    at Socket._onTimeout (net.js:333:8)\n    at tryOnTimeout (timers.js:224:11)\n    at Timer.listOnTimeout (timers.js:198:5)","timestamp":"2016-08-10T05:51:08.682Z"}
{"error":{"name":"MongoError","message":"connection 3 to app.parentplanet.com:27017 timed out"},"level":"error","message":"Error generating response. { MongoError: connection 3 to app.parentplanet.com:27017 timed out\n    at Function.MongoError.create (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/error.js:29:11)\n    at Socket.<anonymous> (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/connection/connection.js:161:20)\n    at Socket.g (events.js:286:16)\n    at emitNone (events.js:86:13)\n    at Socket.emit (events.js:185:7)\n    at Socket._onTimeout (net.js:333:8)\n    at tryOnTimeout (timers.js:224:11)\n    at Timer.listOnTimeout (timers.js:198:5)\n  name: 'MongoError',\n  message: 'connection 3 to app.parentplanet.com:27017 timed out' }","timestamp":"2016-08-10T05:51:36.662Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: connection 3 to app.parentplanet.com:27017 timed out\n    at Function.MongoError.create (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/error.js:29:11)\n    at Socket.<anonymous> (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/connection/connection.js:161:20)\n    at Socket.g (events.js:286:16)\n    at emitNone (events.js:86:13)\n    at Socket.emit (events.js:185:7)\n    at Socket._onTimeout (net.js:333:8)\n    at tryOnTimeout (timers.js:224:11)\n    at Timer.listOnTimeout (timers.js:198:5)\n  name: 'MongoError',\n  message: 'connection 3 to app.parentplanet.com:27017 timed out' } MongoError: connection 3 to app.parentplanet.com:27017 timed out\n    at Function.MongoError.create (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/error.js:29:11)\n    at Socket.<anonymous> (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/connection/connection.js:161:20)\n    at Socket.g (events.js:286:16)\n    at emitNone (events.js:86:13)\n    at Socket.emit (events.js:185:7)\n    at Socket._onTimeout (net.js:333:8)\n    at tryOnTimeout (timers.js:224:11)\n    at Timer.listOnTimeout (timers.js:198:5)","timestamp":"2016-08-10T05:51:36.663Z"}
{"error":{"name":"MongoError","message":"connection 2 to app.parentplanet.com:27017 timed out"},"level":"error","message":"Error generating response. { MongoError: connection 2 to app.parentplanet.com:27017 timed out\n    at Function.MongoError.create (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/error.js:29:11)\n    at Socket.<anonymous> (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/connection/connection.js:161:20)\n    at Socket.g (events.js:286:16)\n    at emitNone (events.js:86:13)\n    at Socket.emit (events.js:185:7)\n    at Socket._onTimeout (net.js:333:8)\n    at tryOnTimeout (timers.js:224:11)\n    at Timer.listOnTimeout (timers.js:198:5)\n  name: 'MongoError',\n  message: 'connection 2 to app.parentplanet.com:27017 timed out' }","timestamp":"2016-08-10T05:52:17.211Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: connection 2 to app.parentplanet.com:27017 timed out\n    at Function.MongoError.create (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/error.js:29:11)\n    at Socket.<anonymous> (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/connection/connection.js:161:20)\n    at Socket.g (events.js:286:16)\n    at emitNone (events.js:86:13)\n    at Socket.emit (events.js:185:7)\n    at Socket._onTimeout (net.js:333:8)\n    at tryOnTimeout (timers.js:224:11)\n    at Timer.listOnTimeout (timers.js:198:5)\n  name: 'MongoError',\n  message: 'connection 2 to app.parentplanet.com:27017 timed out' } MongoError: connection 2 to app.parentplanet.com:27017 timed out\n    at Function.MongoError.create (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/error.js:29:11)\n    at Socket.<anonymous> (/Users/PHUONGNGO/MyProjects/vinasource/dashboard/parse-dashboard/node_modules/mongodb-core/lib/connection/connection.js:161:20)\n    at Socket.g (events.js:286:16)\n    at emitNone (events.js:86:13)\n    at Socket.emit (events.js:185:7)\n    at Socket._onTimeout (net.js:333:8)\n    at tryOnTimeout (timers.js:224:11)\n    at Timer.listOnTimeout (timers.js:198:5)","timestamp":"2016-08-10T05:52:17.212Z"}
